src/menu.cpp

```diff
@@ -4,6 +4,8 @@
 #include "context.h"
 // Include custom debug macros
 #include "flags.h"
+// TODO: comment
```

Comment:
Comment this.

Reply:
addressed in: https://github.com/jCallon/squirt/pull/1/commits/edb88a636d2cf54a42f970d190cd054c04993412